plugins/outputs/cloudwatchlogs/pusher.go

```diff
@@ -63,13 +65,15 @@ type pusher struct {
 	wg                    *sync.WaitGroup
 }
 
-func NewPusher(target Target, service CloudWatchLogsService, flushTimeout time.Duration, retryDuration time.Duration, logger telegraf.Logger, stop <-chan struct{}, wg *sync.WaitGroup) *pusher {
+func NewPusher(region string, target Target, service CloudWatchLogsService, flushTimeout time.Duration, retryDuration time.Duration, logger telegraf.Logger, stop <-chan struct{}, wg *sync.WaitGroup, logSrc logs.LogSrc) *pusher {
```

Comment:
Do we need region as a parameter anymore? I think this was only used for a restriction that has since been removed.

Reply:
Yeah, that makes sense, there's no other uses so will remove